share/completions/userdbctl.fish

```diff
@@ -0,0 +1,29 @@
+# Completions for userdbctl (a part of systemd)
+
+set -l commands_users user groups-of-user
+set -l commands_groups group users-in-group
+set -l commands $commands_users $commands_groups
+set -l services (userdbctl services | string match -r -- '\S+' | string match -v -- 'SERVICE')
```

Comment:
Same as in networkctl, quote this so it's done when needed, isntead of when the completions are loaded.

Reply:
Thanks, done.